eclass/python-utils-r1.eclass

```diff
@@ -1518,6 +1546,17 @@ epytest() {
 		fi
 	fi
 
+	# If we are using hypothesis (require use via EPYTEST_PLUGINS, since
+	# ebuilds may disable autoloading manually) *and* hypothesis-gentoo
+	# is available, use it to disable all health checks, to prevent the tests
+	# from failing randomly under load.
+	if has hypothesis "${EPYTEST_PLUGINS[@]}" &&
+		"${EPYTHON}" -c 'import hypothesis_gentoo' 2>/dev/null &&
+		[[ ! ${HYPOTHESIS_NO_PLUGINS} ]]
```

Comment:
To have short circuit, I would move it as the first check, and it makes it clear enough to also have `&& has hypothesis "${EPYTEST_PLUGINS[@]}"` in the same line

Reply:
I left it last since it's unlikely to happen (it's currently used only in `hypothesis` itself, and there we don't use `EPYTEST_PLUGINS` for it).